diff --git a/dist/index.mjs b/dist/index.mjs
index a7646699fc908b116d51df8911255ef40f3f35ff..5ec7d804ca6c8d1fc595e9e7024209f4accd35ee 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -305,7 +305,7 @@ function useChat({
         const existingData = streamDataRef.current;
         await callChatApi({
           api,
-          body: (_b = experimental_prepareRequestBody == null ? void 0 : experimental_prepareRequestBody({
+          body: (_b = experimental_prepareRequestBody == null ? void 0 : await experimental_prepareRequestBody({
             id: chatId,
             messages: chatMessages,
             requestData: chatRequest.data,
