diff --git a/dist/index.mjs b/dist/index.mjs
index 92a80377692488c4ba8801ce33e7736ad7055e43..2490108f931cbdb53d4d0394818ca71eb8c1bec8 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1844,6 +1844,9 @@ function convertToCoreMessages(messages, options) {
               }
               case "file":
               case "reasoning": {
+                if (blockHasToolInvocations) {
+                  processBlock2();
+                }
                 block.push(part);
                 break;
               }
