diff --git a/dist/index.mjs b/dist/index.mjs
index dcbeb1e797ab452ed68965ab4787d378d0ea754e..94bd204acbd31ccc4bc287b0e2204630624d86fd 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -9396,6 +9396,7 @@ var AbstractChat = class {
     } catch (err) {
       if (err.name === "AbortError") {
         this.setStatus({ status: "ready" });
+        this.onError(err);
         return null;
       }
       if (this.onError && err instanceof Error) {
